tutorials/containers/use-cuda-within-lxd/use-cuda-within-lxd.md

```diff
@@ -82,34 +101,33 @@ blacklist nvidiafb
 blacklist rivatv
 ```
 
-## Install proprietary Nvidia drivers
+## Install proprietary NVIDIA drivers
 Duration: 5:00
 
-With LXD, the host machine handles the drivers and passes the resultant device nodes to the container. But CUDA still expects a local driver installation, and this means we need to install identical versions of both the drivers and CUDA on the host and any LXD containers we deploy.
+With LXD, the host machine handles the drivers and passes the resultant device
+nodes to the container. But CUDA still expects a local driver installation, and
+this means we need to have identical versions of both the drivers and CUDA
+on the host and any LXD containers we deploy.
 
-For this reason, we've opted to use a PPA ([https://launchpad.net/~graphics-drivers/+archive/ubuntu/ppa][ppa]) for the drivers. This simplifies installation across both the host and any container instances you deploy. The alternative is to manually build the drivers directly from Nvidia's website, but this takes more effort and is harder to maintain across multiple deployments.
-
-To add the PPA and update your local package database, enter the following:
-
-```bash
-sudo add-apt-repository --update ppa:graphics-drivers/ppa
-```
+Thankfully LXD 3.0 has a way to keep the two in sync without requiring
+installation of all the NVIDIA packages in both host and container.
 
 You can now use `apt` to install the latest version of the drivers, for example:
 
 ```bash
-apt install nvidia-381
+apt install nvidia-384
 ```
 
-positive
-: Take a look at the PPA, or use `apt search nvidia`, to find the version number of the latest drivers.
+You'll need to reboot your machine after installing the NVIDIA drivers, after
+which you can use the `nvidia-smi` command to check that the driver is
+installed and operating correctly.
 
-You'll need to reboot your machine after installing the Nvidia drivers, after which you can use the `nvidia-smi` command to check that the driver is installed and operating correctly. The output will include details similar to the following:
+The output will include details similar to the following:
 
 ```no-highlight
 Thu May 25 13:32:33 2017
 +-----------------------------------------------------------------------------+
-| NVIDIA-SMI 381.22                 Driver Version: 381.22                    |
+| NVIDIA-SMI 390.77                 Driver Version: 390.77                    |
```

Comment:
You installed `nvidia-384` above and got version `390.77`. is this correct?

Reply:
nvidia-384 is a transitional package which installs nvidia-390 apparently :)

So I've now updated the tutorial to say nvidia-390 instead, will be less confusing.

Reply:
Thanks :+1: